src/static/js/timeslider.js

```diff
@@ -37,9 +37,7 @@ function init() {
     // start the custom js
     if (typeof customStart == "function") customStart();
 
-    //get the padId out of the url
-    var urlParts= document.location.pathname.split("/");
-    padId = decodeURIComponent(urlParts[urlParts.length-2]);
+    padId = window.PADID;
```

Comment:
This is not our naming convention..  What's going on here? :D!

Reply:
Explained in the last [comment](https://github.com/ether/etherpad-lite/pull/4292#issuecomment-700579349)